src/plugins/saved_search/common/types.ts

```diff
@@ -68,6 +68,8 @@ export interface SavedSearchAttributes {
   breakdownField?: string;
   density?: DataGridDensity;
   visContext?: VisContextUnmapped;
+
+  enableFlyout?: boolean;
```

Comment:
Is this used? This type is the saved search data model, so it should only contain persisted properties.

Reply:
No, sorry, just left in from when I was trying to figure out where stuff should go 😅